### PR TITLE
add Browser Alerts

### PR DESCRIPTION
**Pull request by kinseypdx**
# Summary

This PR adds alerts related to the Browser Product including:
* First Input Delay
*  Throughput
* Ajax Throughput
* JS Errors
* Largest Contentful Paint
* Pageload Time

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

**Review by caylahamann — COMMENTED** [review]
Hello! because no browser alert policies existed before this PR, you need to add a reference to the alert-policies in the [`quickstarts/browser/config.yml`](https://github.com/newrelic/newrelic-quickstarts/blob/main/quickstarts/browser/config.yml)

[Here's](https://github.com/newrelic/newrelic-quickstarts/blob/79839b15464195c0705284a3b4a253dc5f77be96/quickstarts/activemq/config.yml#L49-L50) an example of that configuration for `activemq`

**Comment by kinseypdx** [comment]
Thanks for that catch, @caylahamann  - let me know if anything else is missing!

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @kinseypdx, it looks like a few of those alerts are malformed yaml. If you add my suggestions that should get them fixed up!

File: quickstarts/browser/config.yml [hunk 1]
```diff
@@ -43,3 +43,6 @@ dashboards:
   - browser-pages-dashboard
   - javascript-errors
   - traffic-analysis
+alertPolicies:
+  - browser
+  - ajax-example
```
Comment (brammerl) [inline quickstarts/browser/config.yml:48]: Seems like this alert policy doesn't exist. Either remove it or create an alert with this name.

File: alert-policies/browser/AjaxThroughput.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+nrql:
+  query: "SELECT rate(count(*), 1 minute) FROM AjaxRequest WHERE appName like '%' FACET appName"
+baselineDirection: UPPER_AND_LOWER
+terms:
+  - priority: CRITICAL
+    operator: ABOVE
+    threshold: 3
+    thresholdDuration: 500
+    thresholdOccurrences: ALL
+signal:
```
Comment (brammerl) [inline alert-policies/browser/AjaxThroughput.yml:14]: same here

File: alert-policies/browser/FirstInputDelay.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+nrql:
+  query: "SELECT percentile(firstInputDelay, 75) FROM PageViewTiming WHERE appName like '%' FACET appName"
+baselineDirection: UPPER_ONLY
+terms:
+  - priority: CRITICAL
+    operator: ABOVE
+    threshold: 3
+    thresholdDuration: 500
+    thresholdOccurrences: ALL
+signal:
```
Comment (brammerl) [inline alert-policies/browser/FirstInputDelay.yml:14]: also here

File: alert-policies/browser/JSErrors.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  query: "SELECT count(errorMessage) / count(pageUrl) FROM JavaScriptError, PageView  WHERE appName like '%' FACET appName"
+baselineDirection: UPPER_ONLY
+terms:
+  - priority: CRITICAL
+    operator: ABOVE
+    threshold: 3
+    thresholdDuration: 500
+    thresholdOccurrences: ALL
+signal:
+  evaluationOffset: 3
```
Comment (brammerl) [inline alert-policies/browser/JSErrors.yml:15]: also here.

File: alert-policies/browser/AjaxThroughput.yml (hunk 2 above)
Comment (kinseypdx) [reply alert-policies/browser/AjaxThroughput.yml:14]: I removed the `fillOption` and `fillValue` fields and the defaults (`STATIC` and `0`) are OK with me. 👍  
Marking the rest of the comments as resolved as this is the same fields noted across the review!

**Review by brammerl — APPROVED** [review]

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**